x-pack/plugins/maps/public/components/layer_panel/view.js

```diff
@@ -161,7 +165,7 @@ export class LayerPanel  extends React.Component {
         </EuiFlyoutBody>
 
         <EuiFlyoutFooter className="mapLayerPanel__footer">
-          <FlyoutFooter/>
+          <FlyoutFooter hasStateChanged={this.state.hasStateChanged}/>
```

Comment:
this should just be:
`<FlyoutFooter hasStateChanged={this.props.hasStateChanged}/>`

See other comments for why.